### PR TITLE
Rke2 1.22 1.21

### PR DESCRIPTION
**Pull request by briandowns**

### CODE REVIEW

File: channels-rke2.yaml [hunk 1]
```diff
@@ -193,3 +193,47 @@ releases:
       rke2-ingress-nginx:
         repo: rancher-rke2-charts
         version: 4.0.305
+  - version: v1.21.7-rc2+rke2r1
+    minChannelServerVersion: v2.6.1-alpha1
+    maxChannelServerVersion: v2.6.99
+    serverArgs: &serverArgs-v1-21-7-rc2-rke2r1
+      <<: *serverArgs-v1-21-5-rke2r2
+      etcd-arg:
+        type: string
```
Comment (rancher-max) [inline channels-rke2.yaml]:
```suggestion
        type: array
```

File: data/data.json [hunk 2]
```diff
@@ -11438,6 +11438,338 @@
...
+       "rke2-metrics-server"
+      ],
+      "type": "array"
+     },
+     "disable-kube-proxy": {
+      "default": false,
+      "type": "boolean"
+     },
+     "etcd-arg": {
+      "type": "string"
```
Comment (rancher-max) [inline data/data.json:11578]:
```suggestion
      "type": "array"
```

File: data/data.json [hunk 3]
```diff
@@ -11438,6 +11438,338 @@
...
```
Comment (rancher-max) [inline data/data.json:11744]:
```suggestion
      "type": "array"
```

**Review by rancher-max — COMMENTED** [review]
Dang, we have a lot of version bumps 😦 

Just update etcd-arg to type: array and it LGTM

**Review by rancher-max — APPROVED** [review]
LGTM

**Review by dereknola — APPROVED** [review]

**Pull request merged**